Subject: use pkg-config instead of directfb-config
From: Michael Olbrich <m.olbrich@pengutronix.de>

Use pkg-config instead of directfb-config to check for DirectFB.
This will automatically use the special ptxdist pkg-config.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 config.tests/qws/directfb/directfb.pri |    6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

Index: b/config.tests/qws/directfb/directfb.pri
===================================================================
--- a/config.tests/qws/directfb/directfb.pri
+++ b/config.tests/qws/directfb/directfb.pri
@@ -1,7 +1,5 @@
-!cross_compile {
-	DIRECTFBCFLAGS = $$system(directfb-config --cflags)
-	DIRECTFBLIBS = $$system(directfb-config --libs)
-}
+DIRECTFBCFLAGS = $$system(pkg-config directfb --cflags)
+DIRECTFBLIBS = $$system(pkg-config directfb --libs)
 
 QMAKE_CXXFLAGS += $$DIRECTFBCFLAGS
 LIBS += $$DIRECTFBLIBS 
